source/command-logging-and-monitoring/tests/README.rst

```diff
@@ -0,0 +1,59 @@
+.. role:: javascript(code)
+  :language: javascript
+
+==============================
+Command Logging and Monitoring
+==============================
+
+.. contents::
+
+--------
+
+Testing
+=======
+
+Automated Tests
+^^^^^^^^^^^^^^^
+There are tests in the `Unified Test Format <../../unified-test-format/unified-test-format.rst>`__ for both logging and
+monitoring in `/logging </logging>`_ and `/monitoring </monitoring>`_, respectively. Drivers MUST run the logging
```

Comment:
fyi @BorisDog: added a sentence here about increasing the max document length when running these tests.